modules/internal/DefaultRectangular.chpl

```diff
@@ -42,7 +42,10 @@ module DefaultRectangular {
   config param debugDataParNuma = false;
   config param disableArrRealloc = false;
   config param reportInPlaceRealloc = false;
+
   config param parallelAssignThreshold = 2*1024*1024;
+  config param enableParallelGetsInAssignment = false;
+  config param enableParallelPutsInAssignment = false;
```

Comment:
If we wanted a finer grained control, I'd probably do something like:

```chpl
config param localParallelAssignThreshold = 2*1024*1024;
config param getParallelAssignThreshold = -1;
config param putParallelAssignThreshold = -1;
```

to control thresholds individually. I can't imagine any user will use these though, so I'm good punting on that for https://github.com/Cray/chapel-private/issues/1371

Reply:
Ah good point, but having started testing already, I am inclined to punt it.